### PR TITLE
Issue with default role not being used

### PR DESCRIPTION
**Pull request by FilipDusek**
#### What does this PR do?
Fixes a typo causing default role not to be applied

#### How should this be manually tested?
Set a default role with config and check whether it was assigned to requests

### CODE REVIEW

**Comment by nyambati** [comment]
Thanks @FilipDusek  for catching this one, I seem to have not caught this one with a behavior test. Would you mind if you could add a behavior test case. One will suffice

File: lib/nacl.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function config(config, response) {
   opt.response = response;
   opt.baseUrl = options.baseUrl;
   opt.decodedObjectName = options.decodedObjectName;
-  opt.defautRole = options.defautRole || 'guest';
+  opt.defaultRole = options.defautRole || 'guest';
```
Comment (nyambati) [inline lib/nacl.js]: Nice catch here, can you also change `options.defautRole` to `options.defaultRole`

**Comment by FilipDusek** [comment]
@nyambati It sounds interesting and I'd like to try it, but because I'm quite new to this, making behavior test would take me more time than I currently have. 

I expect to have time for this next week. If this is still open by then, I'll make it.

**Comment by nyambati** [comment]
@FilipDusek OK got it.

**Pull request merged**